apps/web/examples/newPage/example.router.js

```diff
@@ -0,0 +1,19 @@
+import { Router as createRouter } from 'express';
+import * as controller from './example.controller.js';
```

Comment:
Are we happy with * imports?  Wouldn't be clearer with named imports such as `import { viewAllExamples, viewExampleById } from controller` or is that another conversation :-) ?

Reply:
This would be a good one to enforce with an eslint rule once we agree a convention

Reply:
Well realistically we would always use all the controllers (otherwise we have controllers that are not used) if that makes sense?